garbage_classifier/garbage_classifier/garbage_data.py

```diff
@@ -34,22 +38,112 @@ def __len__(self):
     
 class GarbageData():
 
-    def __init__(self, dataset_path, batch_size, val_split=0.2):
-        self.dataset_path = dataset_path
+    def __init__(self, train_path, test_path, batch_size):
+        self.train_path = train_path
+        self.test_path = test_path
         self.batch_size = batch_size
         self.feature_extractor = FeatureExtractor('microsoft/beit-base-patch16-224-pt22k-ft22k')
         data_transform_train = transforms.Compose([
             transforms.RandomHorizontalFlip(),
-            transforms.ColorJitter(brightness=0.2, contrast=0.2, saturation=0.1, hue=0.1),
             transforms.RandomAffine(degrees=40, translate=None, scale=(1, 2), shear=15),
             self.feature_extractor])
-        self.data = ImageFolder(self.dataset_path)
-        train_data, validation_data = random_split(self.data, [int(len(self.data) * (1 - val_split)), int(len(self.data) * val_split)])
-        self._train_loader = DataLoader(LazyDataset(train_data, data_transform_train), batch_size=self.batch_size, shuffle=True, num_workers=4)
-        self._val_loader = DataLoader(LazyDataset(validation_data, transforms.Compose([self.feature_extractor])), batch_size=self.batch_size, shuffle=False, num_workers=4)
+        self.train_data = ImageFolder(self.train_path)
+        self.test_data = ImageFolder(self.test_path)
+        self._train_loader = DataLoader(LazyDataset(self.train_data, data_transform_train), batch_size=self.batch_size, shuffle=True, num_workers=4)
+        self._test_loader = DataLoader(LazyDataset(self.test_data, transforms.Compose([self.feature_extractor])), batch_size=self.batch_size, shuffle=False, num_workers=4)
 
     def get_train_loader(self):
         return self._train_loader
 
-    def get_val_loader(self):
-        return self._val_loader
\ No newline at end of file
+    def get_test_loader(self):
+        return self._test_loader
+    
+def load_train_data(s3_access_key, s3_secret_key, s3_bucket, s3_prefix):
+    s3_client = boto3.client('s3', aws_access_key_id=s3_access_key, aws_secret_access_key=s3_secret_key)
+    folder = 'data/real-ds'
+    _download_files_from_s3_bucket_folder(s3_client, s3_bucket, s3_prefix, f'{folder}/input')
+    _create_train_test_split(f'{folder}/input', f'{folder}/train', f'{folder}/test')
+    _create_tar_gz_folder(f'{folder}/train', 'data/real-ds/train.tar.gz')
+    _create_tar_gz_folder(f'{folder}/test', 'data/real-ds/test.tar.gz')
```

Comment:
please remove hardcode, give names

Reply:
Hardcode removed in latest pr for Airflow https://github.com/rosklyar/projector-ml-ops/pull/13/files#diff-602516c0892d0bcf7a23fc1348d4275fdf4ef1d10e4e476ffb7dba3f39990d1dR61